server/protocols/jmap-draft/src/test/java/org/apache/james/jmap/draft/methods/MIMEMessageConverterTest.java

```diff
@@ -19,13 +19,61 @@
 
 package org.apache.james.jmap.draft.methods;
 
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+import java.io.ByteArrayInputStream;
+import java.nio.charset.StandardCharsets;
+import java.sql.Date;
+import java.time.Instant;
+import java.time.ZoneId;
+import java.time.ZonedDateTime;
+
+import org.apache.james.core.Username;
+import org.apache.james.jmap.draft.methods.ValueWithId.MessageWithId;
+import org.apache.james.jmap.draft.model.CreationMessage;
+import org.apache.james.jmap.draft.model.CreationMessage.DraftEmailer;
+import org.apache.james.jmap.draft.model.CreationMessageId;
+import org.apache.james.mailbox.AttachmentContentLoader;
+import org.apache.james.mailbox.MailboxSession;
+import org.apache.james.mailbox.MailboxSessionUtil;
+import org.apache.james.mailbox.model.AttachmentId;
+import org.apache.james.mailbox.model.Cid;
+import org.apache.james.mailbox.model.MessageAttachment;
+import org.apache.james.mime4j.codec.EncoderUtil;
+import org.apache.james.mime4j.codec.EncoderUtil.Usage;
+import org.apache.james.mime4j.dom.Entity;
+import org.apache.james.mime4j.dom.Message;
+import org.apache.james.mime4j.dom.Multipart;
+import org.apache.james.mime4j.dom.TextBody;
+import org.apache.james.mime4j.dom.address.Mailbox;
+import org.apache.james.mime4j.dom.field.ContentTypeField;
+import org.apache.james.mime4j.message.BasicBodyFactory;
+import org.apache.james.mime4j.stream.Field;
+import org.assertj.core.data.Index;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableMap;
+
 class MIMEMessageConverterTest {
+    MailboxSession session;
+    AttachmentContentLoader attachmentContentLoader;
+
+    @BeforeEach
+    void setUp() {
+        session = MailboxSessionUtil.create(Username.of("bob"));
+        attachmentContentLoader = mock(AttachmentContentLoader.class);
```

Comment:
why not a memory implementation?

Reply:
Instanciating a full fledge MemoryAttachmentManager is no easy thing.

This attachment loader is not exercised by these tests, this is a rare usage of mocks my personal ethic authorizes...

Reply:
The level of these tests, halth unit, halth integration would require a full fladge memory test suite making hard isolation testing.

I am not convinced by this.